src/main/java/de/uni_tuebingen/ub/nppm/servlet/backend/ContentServlet.java

```diff
@@ -26,46 +25,62 @@ public class ContentServlet extends AbstractBackendServlet {
     protected void generatePage(HttpServletRequest request, HttpServletResponse response) throws Exception {
         PrintWriter out = response.getWriter();
 
-        // show TinyMCE
-        if (request.getParameter("loadFile") != null) {
-            RequestDispatcher rd = request.getRequestDispatcher("tinyMce.jsp");
-            rd.include(request, response);
+        if (!isAdminRequired(request, response)) {
+            //If the user is not an administrator, redirect them to logout.jsp.
+            RequestDispatcher dispatcher = request.getRequestDispatcher("/logout.jsp");
```

Comment:
This check will be done by the parent function. You should only override the isAdminRequired function.

Reply:
done: look at commit: Administration